rfc-113-expanding-draft-access-for-unauthenticated-users.md

```diff
@@ -0,0 +1,212 @@
+  [auth-bypass token documentation][auth-bypass-token-docs] will be updated to
+  reflect this as the best practice for future auth-bypass tokens
+
+### Rejected alternative approaches
+
+There were a number of alternative ideas which were considered to provide fact
+checking for step-by-steps and were ultimately rejected. This is a quick
+compilation of them and the reasons why they were rejected.
+
+#### Providing a link to preview the embedded part of a step-by-step distinct from the content
+
+This would involve creating a page which shows just the step-by-step
+navigation component that is embedded on linked pages. This was deemed
+insufficient as fact checkers are often comparing the step-by-step navigation
+with the context of the specific guidance pages.
+
+#### Providing a means to embed the step-by-step component into live content
+
+This would involve a means of injecting the step-by-step navigation component
+into live GOV.UK pages via JavaScript. This would prevent a fact checker of a
+step-by-step from viewing any draft content except the step-by-step
+navigation itself. This was quickly rejected as it was considered to be a
+common scenario for the pages that are part of a step-by-step
+to also be drafts.
+
+#### Exclude asset access from fact-checking
+
+Assets are the area of access that provide the most complication since they
+are distinct from the Publishing API link system. An option was to exclude
+them from the access checks and have them return forbidden errors if a
+step-by-step page used them - this would typically be most noticeable on an
+embedded image.
+
+This option was rejected due to it leaving a problem that could be easily
+anticipated and likely having to fall to a support team to resolve or explain
+whenever it occurs.
+
+#### Providing a fact checker access to all resources on the draft stack that are not access limited
+
+An approach that was considered was to allow the fact checker to have access
+to all pages on the draft stack that are not access limited. This would grant
+them similar access to an authenticated user. This offered the advantage that
+it allows simplifying the logic involved in checking tokens and is consistent
+with privacy expectations.
+
+This was rejected since it was a significant departure from the existing access
+allowed to a fact checker through an auth-bypass token and may require
+considerable communication and consultation time. It remains a viable option
+but should be considered as part of a consideration of whether there should be
+more accountability in draft access.
+
+#### Automating the generation of a Heroku preview with a GOV.UK signon account
+
+This would involve using a special GOV.UK signon account to download the pages
+of the step-by-step and then create a small app that can be hosted on Heroku.
+This was rejected due to having a high level of complexity and being
+unconventional compared to other GOV.UK apps.
```

Comment:
There's also a cost involved in hosting Heroku apps, and we sometimes reach the limit of our current subscription (which may be the free tier).